clusters/app.ci/release-controller/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+  - smarterclayton
+  - derekwaynecarr
+  - eparis
+  - jwforres
+  - dptp
+  - sosiouxme
+  - jupierce
+  - bradmwilliams
+reviewers:
+  - dptp
```

Comment:
if you leave dptp here we are back to square one, changes here create notifications for dptp members :) I think its fine to have the same ppl be approvers and reviewers and if dptp input is needed, the github team alias or individual members can be pinged

Reply:
ack. Changed reviewers to art & dp-cr.